# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/fields/basic.js
# Commit: d96f98f974e0
# Full Hash: d96f98f974e087224e0a1ef9be67e2fc63397d8b
# Author: Ashley Hauck <khyperia@mozilla.com>
# Date: 2019-03-21 04:34:45
# Regressor Bug: 1535166
# File Overlap Count: 1
# Description:
#   Bug 1535166 - Implement computed field names. r=jorendorff
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23408
# ==============================================================================

diff -r c786b72a9d8e -r d96f98f974e0 js/src/jit-test/tests/fields/basic.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/fields/basic.js	Wed Mar 20 17:26:01 2019 +0000
@@ -0,0 +1,14 @@
+// |jit-test| --enable-experimental-fields
+
+class C {
+    x;
+    y = 2;
+}
+
+class D {
+    #x;
+    #y = 2;
+}
+
+if (typeof reportCompare === "function")
+  reportCompare(true, true);